gate.lic

```diff
@@ -0,0 +1,194 @@
+custom_require.call(%w[common common-arcana events spellmonitor])
```

Comment:
Would you mind adding a documentation block as a header?

Reply:
done